circle.yml

```diff
@@ -0,0 +1,62 @@
+# Setup in CircleCI account the following ENV variables:
+# BINTRAY_ORGANIZATION
+# BINTRAY_ACCOUNT
+# BINTRAY_API_KEY
+general:
+  # Don't run CI for PR, only for major branches
+  branches:
+    only:
+      - master
+      - /v[0-9]+(\.[0-9]+)*/
+      - feature/circleci
+  artifacts:
+    - ~/packages
+
+machine:
+  environment:
+    ST2_PACKAGES_REPO: https://github.com/StackStorm/st2-packages
+  pre:
+    - mkdir -p ~/packages
+
+checkout:
+  post:
+    - git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2web/st2-packages
+
+dependencies:
+  pre:
+    - sudo apt-get install rpm
+    - npm install bower
+  post:
+    - node_modules/.bin/bower install
+
+test:
+  post:
+    - node_modules/.bin/gulp production
+
+deployment:
+  publish:
+    owner: StackStorm
+    branch:
+      - master
+      - /v[0-9]+(\.[0-9]+)*/
+      - feature/circleci
+    commands:
+      - dpkg-buildpackage -b -uc -us
+      - rpmbuild -bb rpm/st2web.spec
```

Comment:
I think these 2 lines should be part of `test` section.

This will allow to check if packages are able to build if you/or someone open PR against these `master` or `vX.Y.Z` branches.